build/Microsoft.DotNet.Cli.Publish.targets

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(ToolsDir)/PublishContent.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)/publish/Microsoft.DotNet.Cli.Badge.targets" />
+
+  <UsingTask TaskName="UploadToAzure" AssemblyFile="$(BuildToolsTaskDir)/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+  <UsingTask TaskName="FinalizeBuildTask" AssemblyFile="$(CLIBuildDll)" />
+  <UsingTask TaskName="SetBlobPropertiesBasedOnFileTypeTask" AssemblyFile="$(CLIBuildDll)" />
+
+  <!-- PUBLISH_TO_AZURE_BLOB env variable set by CI -->
+  <Target Name="Publish"
+          Condition=" '$(PUBLISH_TO_AZURE_BLOB)' != '' "
+          DependsOnTargets="Prepare;
+                            Package;
+                            PublishArtifacts;
+                            FinalizeBuild" />
+
+  <Target Name="PublishArtifacts" DependsOnTargets="SetupAzureBlobInformation;
+                                                    GenerateVersionBadge;
+                                                    GatherItemsForPattern;
+                                                    UploadToAzure;
```

Comment:
What do you think of adding a comment? I think anyone looking at this file in the future will be confused.